src/server/utilities/liteATITracking/viewTracking/index.ts

```diff
@@ -0,0 +1,20 @@
+export default () => {
+  const options = { threshold: 0.5 };
+  const observer = new IntersectionObserver(entries => {
+    console.log('INTERSECTION OBSERVER LOADED');
+    entries.forEach(entry => {
+      if (entry.isIntersecting) {
+        console.log('IS INTERSECTING');
+        const { target } = entry;
+        const atiURL = target.getAttribute('data-lite-ati-view-tracking');
+        setTimeout(() => {
+          window.processClientDeviceAndSendLite(atiURL as string);
+          observer.unobserve(target);
+        }, 1000);
```

Comment:
Should we also use a const for this timeout - https://github.com/bbc/simorgh/blob/bcd1df464aafb8dcd5949bb2b3227bea9fdcc11f/src/app/hooks/useViewTracker/index.jsx#L11